maestro-orchestra-models/src/main/java/maestro/orchestra/Commands.kt

```diff
@@ -357,9 +380,14 @@ data class LaunchAppCommand(
     val stopApp: Boolean? = null,
     var permissions: Map<String, String>? = null,
     val launchArguments: Map<String, Any>? = null,
+    val label: String? = null,
 ) : Command {
 
     override fun description(): String {
+        if (label != null && label != ""){
```

Comment:
```suggestion
        if (label != null){
```

See my comment on `Condition.kt`